.github/workflows/ci.yml

```diff
@@ -56,6 +56,10 @@ jobs:
       run:
         shell: bash
     steps:
+      - name: Install Rust toolchain
```

Comment:
Not sure why you needed to add this. GHA's default runners now come with the Rust toolchain already. If you simply needed to update, could have done `cargo up` instead. Definitely not blocking though.

Reply:
Yup - removed.